qrenderdoc/Widgets/Extended/RDSplitter.h

```diff
@@ -0,0 +1,78 @@
+#ifndef RDSPLITTER_H
+#define RDSPLITTER_H
```

Comment:
Can you copy-paste the preamble from other similar headers like `RDTreeView.h`? i.e. the full license text (copyright this year) and just `#pragma once`, not using `#ifndef` include guards.

Same applies to the .cpp, except only the license comment, not the `#pragma once` :smile:.